Scripts/Global/Global.gd

```diff
@@ -0,0 +1,18 @@
+extends Node
+
+# --- Public Properties ---
+var map: TileMapLayer       ## Reference to the TileMapLayer node for interacting with the game map.
+var player: PlayerNode      ## Reference to the Player for interacting.
```

Comment:
This can cause issues in the future if we want multiplayer. No problem for now but if we want MP later we're going to want to replace this.

Reply:
it shouldn't unless we send it to the other computers, but I could be wrong as I haven't used multiplayer that much.